src/test/java/com/scalefocus/libraryproject/BasicTests.java

```diff
@@ -0,0 +1,48 @@
+package com.scalefocus.libraryproject;
+
+import com.scalefocus.libraryproject.entities.UserEntity;
+import com.scalefocus.libraryproject.enums.Role;
+import com.scalefocus.libraryproject.models.UserModel;
+import com.scalefocus.libraryproject.repositories.UserRepository;
+import com.scalefocus.libraryproject.services.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+@SpringBootTest
+public class BasicTests {
```

Comment:
Please, rename this test suite to UserServiceTest and provide a UserControllerTest suite.